compiler/rustc_mir_transform/src/unreachable_prop.rs

```diff
@@ -13,7 +13,9 @@ pub struct UnreachablePropagation;
 impl MirPass<'_> for UnreachablePropagation {
     fn is_enabled(&self, sess: &rustc_session::Session) -> bool {
         // Enable only under -Zmir-opt-level=2 as this can make programs less debuggable.
-        sess.mir_opt_level() >= 2
+        // Coverage gets confused by MIR passes that can remove all coverage statements
+        // from an instrumented function.
```

Comment:
Could you add a FIXME with a issue/PR number, so a future reader will know when to revert that PR?
r=me once done

Reply:
I've added a FIXME that refers to a new issue, #116171.